.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!--
+Thank you for your interest in and contributing to ECS! There are a
+few simple things to check before submitting your pull request that
+can help with the review process. You should delete these items from
+our submission, but they are here to help bring them to your attention.
+-->
+
+- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
+- Have you followed the [contributor guidelines](https://github.com/elastic/ecs/blob/master/CONTRIBUTING.md)?
+- If submitting code/script changes, have you verified all tests pass locally using `make test`?
+- If submitting schema/fields updates, have you generated new artifacts by running `make` and committed those changes?
+- Is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.
+- Have you added an entry to the [CHANGELOG](https://github.com/elastic/ecs/blob/master/CHANGELOG.next.md)?
```

Comment:
Perhaps make the name more obvious in the link text here. People sometimes still get confused between the two files
```suggestion
- Have you added an entry to the [CHANGELOG.next.md](https://github.com/elastic/ecs/blob/master/CHANGELOG.next.md)?
```